### PR TITLE
Functionality to shift days and months

### PR DESCRIPTION
**Pull request by Euen**

### CODE REVIEW

File: src/ktn_date.erl [hunk 1]
```diff
@@ -27,3 +31,39 @@ now_human_readable() ->
...
+        false ->
+            error(out_of_bounds)
+    end.
+
+%% @doc Returns `Date' if valid. Otherwise, returns `Date' replacing `Day`
+%% with the last day of the month.
+find_valid_date(Date) ->
+    case calendar:valid_date(Date) of
+      true ->
+         Date;
```
Comment (elbrujohalcon) [inline src/ktn_date.erl]: Incompatible indentation

File: src/ktn_date.erl [hunk 2]
```diff
@@ -27,3 +31,39 @@ now_human_readable() ->
...
+            find_valid_date({Year, Month+1, D});
+        false ->
+            error(out_of_bounds)
+    end.
+
+%% @doc Returns `Date' if valid. Otherwise, returns `Date' replacing `Day`
+%% with the last day of the month.
+find_valid_date(Date) ->
+    case calendar:valid_date(Date) of
+      true ->
```
Comment (elbrujohalcon) [inline src/ktn_date.erl]: Incompatible indentation

**Comment by Euen** [comment]
done

**Comment by elbrujohalcon** [comment]
@Euen can you add a bit of doc to the README, please?

**Comment by Euen** [comment]
@elbrujohalcon README updated, it was that what you meant?

**Pull request merged**